<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SHome</class>
 <widget class="QDialog" name="SHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{background-color:rgb(43, 49, 56);
}
QPushButton
{
border-style:bold;
border-width:0px;
border-color:green;
background-color:rgb(0, 192, 255);
font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
color:white;
border-radius:5px;
}
QPushButton:hover
{
background-color:white;
color:rgb(0, 192, 255);
border-color:green;
border-width:10px;
border-style:bold;
}
QPushButtonpressed
{
color:rgb(255, 255, 0);
background-color:rgb(52, 168, 83);
border-width:2px;}
QLabel
{
font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,192,255);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Student Home&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Tuition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Unenroll</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
color:rgb(12, 211, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(12, 211, 255);
font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-radius:100px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
